tutor/templates/base.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">  
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" 
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+       <title> {% block title %} Tutor {% endblock %} </title>
+    </head>
+    <body style="background: url(../static/img/pexels-lukas-317356.jpg); background-size: cover">  
+
+        <nav class="navbar navbar-expand-lg navbar-light bg-light" style="background: url(../static/img/pexels-lukas-317356.jpg); background-size: cover" >
+          <a href="/"><img src="../static/img/logo.jpeg" width=100 height=50></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+          
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+              <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                  <a class="nav-link" href="#"><mark>Courses</mark></a>
+                </li>
+              </ul>
+              <div class="my-2 my-lg-0">
+                <a class="nav-link" href="#"><mark>Sign up</mark></a>
```

Comment:
I am wondering, why did you choose "mark" here? Is it because of its background? If so, there are better ways to do it.

IMHO, discard the "mark" tags and add styles to "a" adding background and padding to make it looks better.